docs/source/quick_start/troubleshooting.rst

```diff
@@ -0,0 +1,130 @@
+
+Troubleshooting
+===============
+
+Have problems making things work? Here are some common issues and their solutions
+
+MCDR won't start
+----------------
+
+* **mcdreforged: command not found**
+    * Try ``python -m mcdreforged``
+    * If you installed MCDR via pipx, ensure the pipx apps directory is in your PATH
+    * If you installed MCDR via pip globally, ensure the pip scripts directory is in your PATH
```

Comment:
To avoid confusion "ensure the pip executable is in your PATH"

```suggestion
    * If you installed MCDR via pip globally, ensure that the directory where pip places the installed scripts is in your PATH
```